libraries/aws/provisioning/test/aws_iot_tests_provisioning.c

```diff
@@ -44,13 +44,13 @@ void RunProvisioningTests( bool disableNetworkTests,
     /* Silence warnings about unused parameters. */
     ( void ) disableLongTests;
 
-    RUN_TEST_GROUP( Provisioning_Unit_API );
+    /* RUN_TEST_GROUP( Provisioning_Unit_API ); */
```

Comment:
Delete code rather than commenting it out.  Same comment at lines 49/53

Reply:
Ohh, accidentally pushed this. Will revert the change.

Reply:
Will this be reverted as part of this PR?

Reply:
Looks like you still need to remove this commented-out code